packages/create-sitecore-jss/src/templates/nextjs-sxa/src/components/Title.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import {
+  Link,
+  Text,
+  useSitecoreContext,
+  LinkField,
+} from '@sitecore-jss/sitecore-jss-nextjs';
+import {
+  TextField,
```

Comment:
Should be imported from `sitecore-jss-nextjs`, if it's not exported there, add additional export to this package